jira/client.py

```diff
@@ -2828,15 +2830,26 @@ def search_users(
         :type includeActive: bool
         :param includeInactive: If true, then inactive users are included in the results. (Default: False)
         :type includeInactive: bool
+        :param query: Search term. It can just be the email.
+        :type query: str
 
 
         :rtype: ResultList
         """
+        if not user and not query:
+            raise ValueError("Either 'user' or 'query' arguments must be specified.")
+
         params = {
```

Comment:
why not retain existing changes and update params
`
params = {
            "query": user,
            ......
     }
`

Reply:
@ngenovictor if just using the `query` param is backwards compatible, I'm fine with that too

Reply:
I ran tests and query looks to be backward compatible. it works with api version 2 and 3 both.